Dockerfile

```diff
@@ -22,6 +22,13 @@ RUN dotnet build src/Designer/Designer.csproj -c Release -o /app_output
 RUN dotnet publish src/Designer/Designer.csproj -c Release -o /app_output
 RUN rm -f /app_output/Altinn.Studio.Designer.staticwebassets.runtime.json
 
+# Prepare app template
+FROM alpine:3.17.0 as app-template
+WORKDIR /app_template
+RUN apk add jq
+RUN wget -O - https://api.github.com/repos/Altinn/app-template-dotnet/releases/latest | jq '.tarball_url' | xargs wget -O - | tar -xz
+RUN mv Altinn-app-template-dotnet-* AppTemplateRoot ; mv AppTemplateRoot/src/* . ; rm -r AppTemplateRoot
+
```

Comment:
Don't do this as a separate step. Just put it into one of the other ones. I know it seems cleaner to have all those separate steps, but its slower. We should probably just have one build step ideally.

Anyway, not a breaking issue.